File: test_browser.js
const { chromium } = require('playwright');
(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  await page.goto('http://localhost:8000/sim_game.html');
  // wait for it to run
  await page.waitForTimeout(2000);
  const title = await page.title();
  console.log("Page title:", title);
  await browser.close();
})();

File: test_browser.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch()
    page = browser.new_page()
    page.goto('http://localhost:8000/sim_game.html')
    page.wait_for_timeout(2000)
    print("Page title:", page.title())
    browser.close()
